src/client/client.cpp

```diff
@@ -653,8 +669,8 @@ bool Client::loadMedia(const std::string &data, const std::string &filename)
 	};
 	name = removeStringEnd(filename, image_ext);
 	if (!name.empty()) {
-		verbosestream<<"Client: Attempting to load image "
-		<<"file \""<<filename<<"\""<<std::endl;
+		TRACESTREAM(<< "Client: Attempting to load image "
```

Comment:
Just my opinion: Wouldn't it look better to include the first `<<` operator into the `log.h` define?

Reply:
I'm sure it would, but that would be lots of line changes for small benefit.